Controller/SectionController.php

```diff
@@ -46,11 +46,13 @@ public function listAction()
     {
         $sectionList = $this->sectionService->loadSections();
         $contentCountBySectionId = [];
+        $canDeleteBySectionId = [];
         $deleteFormsBySectionId = [];
 
         foreach ($sectionList as $section) {
             $sectionId = $section->id;
             $contentCountBySectionId[$sectionId] = $this->sectionService->countAssignedContents($section);
+            $canDeleteBySectionId[$sectionId] = !$this->sectionService->isSectionUsed($section);
```

Comment:
also wondering if going this way won't lead us to have too much lines of code here and even too much calls... (actually 3 per loop).
you wouldn't expect have lot of sessions here, but who knows...